Minutes/2023/2023_08_07-minutes.md

```diff
@@ -0,0 +1,211 @@
+
+Ege Korkan: that's sort of workflow and we are recently starting on a like a full website.
+
+Ege Korkan: So that the the tutorial content can be read at the same time so that it's not just the the videos you have to watch like by stopping what you're doing  and so on and so forth.
+
+Ege Korkan: you said you  have interest on semantic web parts or maybe some other things?
+
+Ege Korkan: I think we are also new on this kind of workflows and like since it's not like code reviews or it's not simple text files to work.
+
+Justus Fries: But the the mode of operation is still over GitHub, right?
+
+Ege Korkan: Yes. So the the files will be all there, like all the PowerPoint files, text for the audio and final video.
+
+Ege Korkan: the website will be also on GitHub.
+
+Ege Korkan: We also talked with some people from W3C and it might be possible to push it to a W3C developer's page and which is generally more HTML, CSS kind of tutorials together with edX.
+
+Ege Korkan: you can certificate of attendance.
+
+Justus Fries: so it is about showing trust of attendance?
+
+Cristiano Aguzzi: yes, I am sending a link.
+
+Ege Korkan: So for that we need to generate more content first to to show how it really looks like and something that can be certified because the single video or tutorial lighting at the moment would be too short.
+
+Justus Fries: ok
+
+Justus Fries: So and right now you have like what I saw on GitHub the the introduction right that does like the basic overview of what you can like do with it.
+
+Justus Fries: And so is there anything else that you have already finished or are you still working on? Like making it deeper?
+
+Idil Sezgin: actually published first two videos in the GitHub if I'm not mistaken.
+
+Idil Sezgin: First one is introduction to this series. Basically like what you need to expect from the series.
+
+Idil Sezgin: Like what is web of things.
+
+Idil Sezgin: we have 12 right now.
+
+Idil Sezgin: So basically, for the first six one we explained JSON and JSON schema.
+
+Idil Sezgin: Both in practice and the as a concept, and then we moved to the OSI layers and then we explain general generic web of things.
+
+Idil Sezgin: I'm still working on the new videos.
+
+Idil Sezgin: So for the website, I hope to have a first version of how it will look like by tpac.
+
+Ege Korkan: So we can actually add more things to preliminary stuff like you mentioned and think of a way how we can like work together.
+
+Ege Korkan: So you can provide the script or do the points and so on like what a voice over should generate.
+
+Justus Fries: How deep does it go. I guess the closest technology wise is kind of the JSON schema tutorial.
+
+> Ege Korkan: shows the slides
+
+Ege Korkan: And so like, it explains the concept like very basically and like the the simple types but I think we can also even go into like type generation like because we actually use it for an scripting API for type generation
```

Comment:
```suggestion
Ege Korkan: it explains the concept very basically and like the simple types but I think we can also even go into type generation like because we actually use it for scripting API for type generation
```